Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme

```diff
@@ -23,7 +23,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
Do we need to switch a build configuration to Release?

Reply:
Isn't this one of the goals of this PR?

Reply:
Fixed with 46a2635